README.md

```diff
@@ -169,6 +169,7 @@ There is 2 different types of buffer (filetype): **klist** and **kinfo** (for ta
 | Stop | `<S>` | Stop the task under cursor |
 | Toggle | `<Enter>`, `<t>` |  Start or stop the task under cursor |
 | Done | `<D>` | Mark task under cursor as done |
+| UnDone | `<U>` | Mark task under cursor as undone |
```

Comment:
It's a detail, but I think it's better to name it `Undone` instead of `UnDone`, since undone is a full word. I would use CamelCase when it's composed of more than one word (for eg. `ToggleHideDone`). What do you think ?

Reply:
Sounds good. I will change all instances of UnDone to Undone.